Intersect.Editor/Localization/Strings.cs

```diff
@@ -4301,15 +4301,6 @@ public struct Tiles
 
             public static LocalizedString layer = @"Layer:";
 
-            public static Dictionary<int, LocalizedString> layers = new Dictionary<int, LocalizedString>
-            {
-                {0, @"Ground"},
-                {1, @"Mask"},
-                {2, @"Mask 2"},
-                {3, @"Fringe"},
-                {4, @"Fringe 2"},
-            };
-
```

Comment:
I recall Panda wanting me to NOT have unlocalizable strings in the config, so I don't think this should be removed but changed instead?

Reply:
So is the goal right now to have a Dictionary in localization of Config String -> Displayed String?  I guess I could do that.